zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/descriptors/TemplateDetails.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.schema.descriptors;
+
+public interface TemplateDetails {
+  String getTemplateName();
+
+  /**
+   * Maps to the create query parameter, if true the corresponding request cannot replace or update
+   * an existing component template, defaults to false
+   *
+   * @return The component template request is create only
+   */
+  default Boolean create() {
```

Comment:
I don't understand how is this going to be used

Reply:
The old operate client uses a custom `overwrite` when making index template requests, overwrite=true means you make the request and it overwrites the current template, overwrite=false means you only make the request if the template doesn't already exist.

This is the exact same behaviour as create, create=true == overwrite=false and vice versa.